content/docs/user-guide/project-structure/dvc-files.md

```diff
@@ -0,0 +1,80 @@
+# `.dvc` Files
+
+You can use `dvc add` to track data files or directories located in your current
+<abbr>workspace</abbr>, or in supported
+[external locations](/doc/user-guide/managing-external-data). Additionally,
+`dvc import` and `dvc import-url` let you bring data from external locations to
+your project, and start tracking it locally.
+
+> See [Data Versioning](/doc/start/data-versioning) and
+> [Data Access](/doc/start/data-access) for more info.
+
+Files ending with the `.dvc` extension ("dot DVC file") are created by these
+commands as data placeholders that can be versioned with Git. They contain the
+information needed to track the target data over time. Here's an example:
+
+```yaml
+outs:
+  - md5: a304afb96060aad90176268345e10355
+    path: data.xml
+    desc: Cats and dogs dataset
+
+# Comments and user metadata are supported.
+meta:
+  name: 'Devee Bird'
+  email: devee@dvc.org
+```
+
+These files use the [YAML 1.2](https://yaml.org/) file format, and a
+human-friendly schema described below. We encourage you to get familiar with it
+so you may modify, write, or generate `.dvc` files on your own.
+
+> See also
+> [How to Merge Conflicts](/doc/user-guide/how-to/merge-conflicts#dvc-files).
+
+## Specification
```

Comment:
let's make it General Spec, and other parts make H2 so they are visible in the left sidebar

Reply:
Good call, done. Not sure about the title though. I went for **Specification (file-level)** for now. WDYT?

Reply:
`Specification (file-level) ` there should be a term for this - something like File schema or File syntax

Reply:
What I'm trying to indicate with "file-level" is that the section focuses on the root fields, subfields are explained in further sections. For now I went back to just **Specification** and added a note about what the section covers.

I also think keeping the word "Specification" is desirable (same as in the dvc.yaml guide) but no strong opinion. "File schema" (in both places) can also work (we do already use the term "schema" in the text).